<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_Dialog</class>
 <widget class="QDialog" name="login_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登入窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhExclusiveInputMask</set>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1" rowspan="3">
    <widget class="QGraphicsView" name="xckj_graphicsView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>490</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/qrc/image/qrc/image/face.png);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QPushButton" name="user_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 204, 255);
subcontrol-position: center;
subcontrol-origin: padding;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>用户管理</string>
       </property>
       <property name="icon">
        <iconset resource="kj.qrc">
         <normaloff>:/qrc/image/qrc/image/modify_icon.png</normaloff>:/qrc/image/qrc/image/modify_icon.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="login_pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 204, 255);
subcontrol-position: center;
subcontrol-origin: padding;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>登入</string>
       </property>
       <property name="icon">
        <iconset resource="kj.qrc">
         <normaloff>:/qrc/image/qrc/image/gotocell.png</normaloff>:/qrc/image/qrc/image/gotocell.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="dataTrans_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 204, 255);
subcontrol-position: center;
subcontrol-origin: padding;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>数据管理</string>
       </property>
       <property name="icon">
        <iconset resource="kj.qrc">
         <normaloff>:/qrc/image/qrc/image/data_icon.png</normaloff>:/qrc/image/qrc/image/data_icon.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="index_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 204, 255);
subcontrol-position: center;
subcontrol-origin: padding;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>指标管理</string>
       </property>
       <property name="icon">
        <iconset resource="kj.qrc">
         <normaloff>:/qrc/image/qrc/image/icon.png</normaloff>:/qrc/image/qrc/image/icon.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="logout_pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 204, 255);
subcontrol-position: center;
subcontrol-origin: padding;
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset resource="kj.qrc">
         <normaloff>:/qrc/image/qrc/image/exit_icon.png</normaloff>:/qrc/image/qrc/image/exit_icon.png</iconset>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;密码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_lineEdit">
       <property name="styleSheet">
        <string notr="true">     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;

background: lightblue</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="styleSheet">
        <string notr="true">     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
background: lightblue</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="hint_pushButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>提示</string>
     </property>
     <property name="icon">
      <iconset resource="kj.qrc">
       <normaloff>:/qrc/image/qrc/image/passwd_icon.png</normaloff>:/qrc/image/qrc/image/passwd_icon.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QLabel" name="title_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Lucida Grande UI'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; color:#0080ff;&quot;&gt;什刹海开景指数系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="hint_label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>login_pushButton</tabstop>
  <tabstop>logout_pushButton</tabstop>
  <tabstop>user_pushButton</tabstop>
  <tabstop>dataTrans_pushButton</tabstop>
  <tabstop>index_pushButton</tabstop>
  <tabstop>hint_pushButton</tabstop>
  <tabstop>xckj_graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="kj.qrc"/>
 </resources>
 <connections/>
</ui>
